lib/matplotlib/scale.py

```diff
@@ -93,7 +93,7 @@ class LogTransformBase(Transform):
     is_separable = True
     has_inverse = True
 
-    def __init__(self, nonpos):
+    def __init__(self, nonpos='mask'):
```

Comment:
This should probably default to 'clip' as this is the default else where.

Reply:
You are right, LogScale uses clip as default and so should LogTransform. Fixed in the PR.